src/test/ui/parser/type-alias-where.stderr

```diff
@@ -1,40 +1,34 @@
-error: where clause not allowed here
+error: where clauses are not allowed after the type for type aliases
   --> $DIR/type-alias-where.rs:8:15
    |
 LL | type Bar = () where u32: Copy;
-   |         -     ^^^^^^^^^^^^^^^
-   |         |
-   |         help: move it here: `where u32: Copy`
+   |               ^^^^^^^^^^^^^^^
 
-error: where clause not allowed here
+error: where clauses are not allowed after the type for type aliases
   --> $DIR/type-alias-where.rs:10:15
    |
 LL | type Baz = () where;
    |               ^^^^^
 
-error: where clause not allowed here
-  --> $DIR/type-alias-where.rs:24:38
+warning: where clause not allowed here
+  --> $DIR/type-alias-where.rs:22:16
    |
-LL |     type Assoc2 where u32: Copy = () where i32: Copy;
-   |                                -     ^^^^^^^^^^^^^^^
-   |                                |
-   |                                help: move it here: `, i32: Copy`
+LL |     type Assoc where u32: Copy = ();
+   |                ^^^^^^^^^^^^^^^     - help: move it here: `where u32: Copy`
```

Comment:
can we add a `.fixed` test?

Reply:
I think maybe you have to add `// run-rustfix` or something like this, you may alsonewed to factor this case out into its own test